packages/ckeditor5-utils/tests/language.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import { getLanguageDirection } from '../src/language';
+
+describe( 'language', () => {
+	describe( 'getLanguageDirection', () => {
+		it( 'determines the language direction', () => {
+			expect( getLanguageDirection( 'en' ) ).to.eql( 'ltr' );
```

Comment:
```suggestion
			expect( getLanguageDirection( 'en' ) ).to.equal( 'ltr' );
```

No idea what's the difference, but we use the longer form everywhere else.